Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -66,6 +67,13 @@ void enableNetworkConnectionIntegrity(NSMutableURLRequest *) { }
 
 namespace WebKit {
 
+static NSString *lastRemoteIPAddress(NSURLSessionDataTask *task)
+{
+    // FIXME: In a future patch, this should adopt CFNetwork API that retrieves the original IP
```

Comment:
Could you file a bug and link to it here, please?

Reply:
Filed: https://bugs.webkit.org/show_bug.cgi?id=246428.